### PR TITLE
Add to Workshop and Onsite Training

### PR DESCRIPTION
**Pull request by weg2g**
Add Coding Bootcamps to workshop and onsite training

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -635,7 +635,7 @@
             "rev": true,
             "title": "Angular Concepts, Code and Collective Wisdom",
             "url": "https://www.udemy.com/angular-2-concepts-code-and-collective-wisdom/"
-          },
+          },
```
Comment (StephenFluin) [inline aio/content/marketing/resources.json]: This looks like an unrelated whitespace change that should be removed?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
I am not seeing Angular training on https://coding-bootcamps.com/. I do see AngularJS though.  This training also doesn't appear to be Angular specific, so will likely need to reject it. (We don't list other general bootcamps/schools that include Angular).

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (weg2g) [reply aio/content/marketing/resources.json]: We have many pages. We offer specific 30 hour course for Angular. See below landing page:
https://coding-bootcamps.com/course_info.php?urlfriendly=website-design-using-Angular-JS-for-beginners

Comment (StephenFluin) [reply aio/content/marketing/resources.json]: It looks like this content is for AngularJS. On our angular.io site we only list Angular content. If this is just a brand mixup, see our [presskit](https://angular.io/presskit) for more information about naming.  which we would [require](https://github.com/angular/angular/blob/master/aio/content/marketing/resources-contributing.md) before listing it.

Comment (weg2g) [reply aio/content/marketing/resources.json]: Also, see Angular training for high school students:
https://myhsts.org/course-website-design-using-Angular-JS-for-beginners.php

Comment (weg2g) [reply aio/content/marketing/resources.json]: I got confused. I want to add High School Technology services (Offering angular to high school students) and Coding Bootcamps (offering angular to adult) to your below resource page:
https://angular.io/resources

I see companies similar to us like https://www.accelebrate.com/ and https://www.webagesolutions.com are listed there. 

Did u get my point?

**Comment by StephenFluin** [comment]
We don't feature AngularJS content on this site. Each of the other onsite trainers listed have met all of our requirements (see [here](https://github.com/angular/angular/blob/master/aio/content/marketing/resources-contributing.md)).

This listing currently fails the following criteria:
* Your contribution must be valid, and contain a link to a page talking specifically about using Angular
* Your resource should follow our brand guidelines (see our Presskit)
* Your resource should already have traction and praise from Angular developers
